admin/tests/integration/components/administration/organizations-import_test.js

```diff
@@ -0,0 +1,69 @@
+import { module, test } from 'qunit';
+import { render } from '@1024pix/ember-testing-library';
+import hbs from 'htmlbars-inline-precompile';
+import { triggerEvent } from '@ember/test-helpers';
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
+import sinon from 'sinon';
+import Service from '@ember/service';
+import setupIntlRenderingTest from '../../../helpers/setup-intl-rendering';
+
+module('Integration | Component |  administration/organizations-import', function (hooks) {
+  setupIntlRenderingTest(hooks);
+  setupMirage(hooks);
+
+  test('it should be possible to create multiple organizations with csv import', async function (assert) {
+    // given
+    const file = new Blob(['foo'], { type: `valid-file` });
+    const notificationSuccessStub = sinon.stub();
+    class NotificationsStub extends Service {
+      success = notificationSuccessStub;
+      clearAll = sinon.stub();
+    }
+    this.owner.register('service:notifications', NotificationsStub);
+
+    // when
+    const screen = await render(hbs`<Administration::OrganizationsImport />`);
+    const input = await screen.findByLabelText(
+      this.intl.t('components.administration.organizations-import.upload-button')
+    );
```

Comment:
ça fonctionne en l'état mais on pourrait peut être avoir un findByRole plutôt, pour préciser qu'il s'agisse d'un bouton ? 

```suggestion
    const input = await screen.findByRole('button', { name:
      this.intl.t('components.administration.organizations-import.upload-button' })
    );
```

Reply:
`findByRole` ou `getByRole` ne va retourner que le label associé à l'input de type file.

`findByLabelText` va retourner uniquement l'input associé au label.

Reply:
C'est bizarre, dans les deux cas on devrait récupérer l'input non ?